upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -107,20 +108,128 @@ backup of Workload Manager configuration data and files is created. Once complet
+With BICAN, the pre-1.2 CAN network is split into two separate networks:
+
+1. Customer Management Network (CMN)
+
+   This network allows only system administrative access from the customer site. The pre-1.2 CAN is renamed to CMN. By
+   the end of the CSM 1.2 upgrade, all non-administrative access, such as from UANs, will be removed from CMN.
+
+   During the CSM 1.2 upgrade, UANs will retain their pre-1.2 CAN IPs in order to minimize disruption to UANs. However,
+   toward the end of the CSM 1.2 upgrade, UANs will stop registering themselves on CMN and will receive new IPs on the
+   CAN/CHN network below. This process is described in more detail in [UAN Migration](#uan-migration).
+
+1. Customer Access Network (CAN) / Customer High-speed Network (CHN)
+
+   For user traffic only (e.g. users running and monitoring jobs), CSM 1.2 allows you to pick one of two networks:
+
+      - Customer Access Network (CAN): this is a new network (VLAN6 in switches) that runs over the management network. This
+         network must not be confused with pre-1.2 CAN, which was a monolithic network that allowed both user and administrative
+         traffic, was configured as VLAN7 in switches, and is now renamed to CMN. The new CAN allows only user traffic.
+
+      - Customer High-speed Network (CHN): this is a new network (VLAN5 in switches) that runs over the high-speed fabric.
+
+   You must only pick either the new CAN or CHN, but not both. The rest of the installation guide will provide you with
+   options for configuring either the new CAN or CHN.
+
+Pivoting the pre-1.2 CAN to the new CMN allows administrative traffic (already on the pre-1.2 CAN) to remain as-is while
+moving standard user traffic to a new site-routable network.
+
+### UAN migration
+
+Certain steps are taken in order to minimize disruption to UANs during the CSM 1.2 upgrade process:
+
+1. During the upgrade, the switch `1.2 Preconfig` will not remove UAN ports from the CMN VLAN (the pre-1.2 CAN), allowing UANs
+   to retain their existing IPs during the CSM 1.2 upgrade process. Traffic to and from UANs will still flow through CMN, but
+   may also flow through CAN/CHN networks if desired.
+
+1. CFS will be temporarily disabled for UANs, so that running CFS plays does not remove CMN interfaces from UANs. As mentioned
+   in [Abstract (Stage 0.3)](#abstract-stage-03), network configuration is controlled by data in SLS, but CFS plays also pick up
+   the same SLS data, which can lead to UANs being prematurely removed from CMN and causing UAN outage. As such, CFS plays
+   need to be disabled for UANs.
+
+   To disable CFS plays for UANs, you must remove CFS assignment for UANs by running the following command:
+
+   ```bash
+   what-is-the-command-to-disable-CFS-for-UANs.sh
+   ```
+
+1. UAN reboots must be avoided. Rebooting a UAN can re-enable CFS and can ultimately lead to removing CMN interfaces from UANs.
```

Comment:
suggest `must be avoided and is not supported`.

Reply:
Thanks. Changed to `UAN reboots must be avoided and is not a supported operation during CSM 1.2 upgrade.`